src/utils/validation/validation.ts

```diff
@@ -1338,6 +1351,36 @@ export function validateGroup(groupId: string, state: IRuntimeState, onlyInRowIn
   );
 }
 
+/*
+ * Validates a repeating groups component specific validations
+ * @param node the node to validate
+ * @param language the current language
+ */
+export function validateRepeatingGroup(node, language: ILanguage): IComponentValidations {
+  const validations: IComponentBindingValidation = { errors: [] };
+  // check if minCount is less than visible rows
+  const repeatingGroupComponent = node.item;
+  const repeatingGroupMinCount = repeatingGroupComponent.minCount || 0;
+  const repeatingGroupVisibleRows = repeatingGroupComponent.rows.filter(
+    (row) => !row.groupExpressions?.hiddenRow,
+  ).length;
+
+  const repeatingGroupMinCountValid = repeatingGroupMinCount <= repeatingGroupVisibleRows;
+
+  // if not valid, return appropriate error message
+  if (!repeatingGroupMinCountValid) {
+    const errorMessage = getParsedLanguageFromKey(
+      'validation_errors.minItems',
+      language,
+      [repeatingGroupMinCount],
+      true,
+    );
```

Comment:
This needs refactoring to `useLanguage`/`langTools` if you merge from main now

Reply:
Will do! 👍